benchmarks/python/rope_ops.py

```diff
@@ -10,6 +10,16 @@
 
 from .model_configs import configs
 
+SEQ_LENGTHS = (
+    None,
+    1024,
+    2048,
+    4096,
+    8192,
+    12288,
+    16384,
+)
```

Comment:
Used the sequence length from issue #4171

Reply:
I just showed part of the results using @kevinstephano's script, but his script covers up to 32K. (https://docs.google.com/document/d/1zJL8zaJJFuK6o0hITJPUZzKSs3sjZzkdncoxsD8G0Nk/edit?tab=t.0)